app/workers/email_archive_worker.rb

```diff
@@ -24,47 +24,34 @@ def perform
 
   def archive_batch(batch)
     archived_at = Time.zone.now
+    exported_at = nil
 
-    import_batch(batch, archived_at)
-    Email.where(id: batch.pluck(:id)).update_all(archived_at: archived_at)
-  end
+    if ENV["EMAIL_ARCHIVE_S3_ENABLED"]
```

Comment:
Might be better to do `.include?` here since `EMAIL_ARCHIVE_S3_ENABLED` with a value of `""` might be confusing.

Reply:
Good point, I'll switch to that.